### ISSUE TITLE
Bug: FT8's first qso not logging properly

### ISSUE THREAD

**Issue by afarhan** [sbitxdev/sbitx#6]
Gyula and two others have reported this. It is a bummer to lose a good contact just because this doesn't work. This is probably the last of the bugs in the logger that will make it useful on FT8.

**Comment by afarhan** [comment]
fixed

**Comment by n1ai** [comment]
Please show which code commit fixed the problem instead of just saying 'fixed'.

**Comment by afarhan** [comment]
Just pull the latest.

**Comment by n1ai** [comment]
"Just pull the latest" is the wrong approach to community software development.

You need to be able to show the commit that fixed the bug and/or the exact lines that were changed to fix the bug.

This helps others learn more about how the code works and perhaps how to make future improvements.

**Comment by afarhan** [comment]
You are right. I should push branches of each issue on sbitxdev and have it
reviewed.
